src/main/java/org/broadinstitute/dsm/route/BSPKitQueryRoute.java

```diff
@@ -138,15 +137,15 @@ public Object getBSPKitInfo(Connection conn, @NonNull String kitLabel, @NonNull
 
     }
 
-    private void writeSampleReceivedToES(DDPInstance ddpInstance, BSPKitQueryResult bspKitInfo) {
+    private static void writeSampleReceivedToES(DDPInstance ddpInstance, BSPKitQueryResult bspKitInfo) {
```

Comment:
result of making instance methods static, Java is designed to be used for object oriented programming, static methods usually breaks all the principles of OOP, in OOP static methods are usually used as utility methods, which calculates something and returns new result instead of mutation.